acceptance-tests/dsl/build.gradle

```diff
@@ -16,6 +16,7 @@ dependencies {
   implementation project(':metrics:core')
   implementation project(':plugin-api')
   implementation project(':plugins:rocksdb')
+  implementation project(':plugins:localfile-securitymodule')
```

Comment:
nit: is there a standard as when we use - or not? i.e. local-file-security-module"? or localfilesecuritymodule?

Reply:
I think this might be the first one which would have three syllables :)